src/components/SidebarNav.vue

```diff
@@ -22,6 +22,11 @@
                         >Getting Started</router-link
                     >
                 </li>
+                <li>
+                    <router-link to="/design-principles"
+                        >Design Principles Toolkit</router-link
```

Comment:
The latest code on the `main` branch has a simpler pattern for adding links to the sidebar. The PR in its current form has a merge conflict, which I think prevented the lookaside deployment from happening.

I can probably fix this merge conflict manually, but going forward it's always helpful to pull the latest from `main` before creating a pull request!